book/homework/new-list.md

```diff
@@ -45,6 +45,10 @@ const addLadyDev = function (ladyDev) {
 </details>
    {% endhint %}
 
+   {% hint style='danger' %}
+   If you didn't use the help section, you'll want to rename the function you created in this step to `addLadyDev` and the parameter to `ladyDev` before continuing.
+   {% endhint %}

```

Comment:
![callout](https://user-images.githubusercontent.com/1417250/57180618-10cfbe80-6e50-11e9-9309-7bae4020d276.PNG)

Reply:
Open to other suggestions. Or even making the attendee troubleshoot it themselves 🤷‍♀

Reply:
IMO I think we can leave it out. I bet most attendees will use the hint section (either because they need guidance or because they'll want to double check). I'll comment it out for now and see how the mentors respond.